File: python/7 kyu/439.py
# https://www.codewars.com/kata/62eb800ba29959001c07dfee/train/python

# Passed

def brightest(colors):
    hex_to_decimal = lambda hex: int(hex, 16) 
    result = max(colors, key=lambda color: max(hex_to_decimal(color[idx:idx+2]) for idx in range(1, 7, 2)))
    return result

output = brightest(["#FFFFFF", "#1234FF"])
print(output)

File: nodejs/7 kyu/439.js
// https://www.codewars.com/kata/594adadee075005308000122/train/javascript

// Passed

function evenAndOdd(num) {
  let digits = String(num)
    .split("")
    .map((strDigit) => Number(strDigit));
  let [evenStrNum, oddStrNum] = ["", ""];
  for (const digit of digits) {
    if (digit % 2 === 0) {
      evenStrNum += String(digit);
    } else {
      oddStrNum += String(digit);
    }
  }

  let [evenNum, oddNum] = [Number(evenStrNum), Number(oddStrNum)];
  let result = [evenNum, oddNum];
  return result;
}

const output = evenAndOdd(126453);
console.log(output);